configfiles.rst

```diff
@@ -278,6 +278,26 @@ mode, with the following types:
     filesystems in setuid-root mode, both inside and outside of SIF files.
     For security reasons, unlike the others this defaults to ``no``.
 
+Disabling Kernel Filesystem Mounts
+==================================
+
+When running in setuid mode, {Project} will mount extfs and squashfs
+filesystems using the kernel's filesystem drivers. These mounts are performed
+for standalone or SIF container images, overlay images or partitions,
+that use extfs or squashfs formats.
+
+Options in ``{command}.conf`` allow these mounts to be disabled, to e.g. work
+around a kernel vulnerability that cannot be patched in a timely manner. Note
+that disabling kernel mounts will result in a significant loss of functionality
+in setuid mode.
+
+``allow kernel squashfs``: Defaults to yes. When set to no, {Project} will not
+mount squashfs filesystems using the kernel squashfs driver.
+
+``allow kernel extfs``: Defaults to yes. When set to no, {Project} will not
+mount extfs filesystems using the kernel extfs driver.
+
+
```

Comment:
Leave this out, we have different options for that functionality, already documented